_security-plugin/configuration/multi-auth.md

```diff
@@ -0,0 +1,130 @@
+---
+layout: default
+title: Multiple option authentication
```

Comment:
Re: your comment in the Asana task, this would need to be "Multiple-option authentication", but rather than naming the "feature" as such, could we just say something like "Multiple authentication options" or "Multiple options for authentication"? Apply the change in language globally.